internal/cmd/protoc-gen-go-flipt-sdk/http.go

```diff
@@ -289,7 +289,7 @@ func generateHTTPMethod(g *protogen.GeneratedFile, m mappings, method *protogen.
 		return
 	}
 
-	g.P("if err := ", protojson("Unmarshal"), "(respData, &output); err != nil { return nil, err }")
+	g.P("if err := ", "(protojson.UnmarshalOptions{DiscardUnknown: true}).Unmarshal", "(respData, &output); err != nil { return nil, err }")
```

Comment:
```suggestion
	g.P("if err := (", protojson("UnmarshalOptions"), "{DiscardUnknown: true}).Unmarshal", "(respData, &output); err != nil { return nil, err }")
```

Reply:
This might seem a little weird. However, it will just ensure that `protojson` is always imported appropriately at the top of the file.

It is being imported elsewhere at the moment, which is why this isn't breaking.
If that were to change those, this could break. So we make sure to use the function here when referencing the type.

Reply:
I should have spent more time trying to figure out how to do it with the `protojson` import 😓 haha thanks. Will make that change right now.

Reply:
No worries :D It is a weird thing `protogen`. Takes some getting used to.